docs/Language Feature Status.md

```diff
@@ -6,6 +6,21 @@ and will be updated as work progresses, features are added / removed, and as wor
 This is not an exhaustive list of our features but rather the ones which have active development
 efforts behind them.
 
+# C# 7.3
+
+| Feature | Branch | State | Developers | Reviewer | LDM Champ |
+| ------- | ------ | ----- | ---------- | -------- | --------- |
+| [Ranges](https://github.com/dotnet/roslyn/blob/features/range/docs/features/range.md) | [range](https://github.com/dotnet/roslyn/tree/features/range) | Prototype | [Ashley](https://github.com/khyperia) | [agocke](https://github.com/agocke) | [jaredpar](https://github.com/jaredpar) |
+
+
+# C# 8.0
```

Comment:
Consider flipping the order around (like a blog): C# 8, 7.3, 7.2, 7.1, ...

Reply:
Wanted to keep the "nearest" release on top which is why I had 7.3.